### PR TITLE
Add Mobile Safari up to 15.4

### PR DESCRIPTION
**Pull request by willyaranda**

### CODE REVIEW

**Review by mimmi20 — CHANGES_REQUESTED** [review]
Please add one unit test or more

**Review by mimmi20 — COMMENTED** [review]
Do these new versions exist also for the non-mobile Safari? If yes, please add the versions for the non-mobile Safari too.

**Comment by willyaranda** [comment]
@mimmi20 sorry, I was working on rebuilding my setup as there was a bump to PHP8 :) 

- I have added the tests
- The Desktop Safaris had this versions added, but not on mobile

File: resources/user-agents/browsers/mobile-safari/mobile-safari-15-x.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
             "iPod": "iPod Touch"
           },
           "platforms": [
-            "iOS_A_15_2", "iOS_A_15_1", "iOS_A_15_0",
+            "iOS_A_15_4", "iOS_A_15_3", "iOS_A_15_2", "iOS_A_15_1", "iOS_A_15_0",
```
Comment (mimmi20) [inline resources/user-agents/browsers/mobile-safari/mobile-safari-15-x.json:27]: This results in the error : 
`Browscap ini file could not be built in UserAgentsTest\V4\FullTest test class, there was an uncaught exception: OutOfBoundsException (Platform "iOS_A_15_4" does not exist in data)`

Please check if these platforms are set in platforms.json file

**Review by mimmi20 — CHANGES_REQUESTED** [review]

File: resources/user-agents/browsers/mobile-safari/mobile-safari-15-x.json (hunk 1 above)
Comment (willyaranda) [reply resources/user-agents/browsers/mobile-safari/mobile-safari-15-x.json:27]: Interesting, I do not get this error when running

`phpunit -c tests/phpunit-integration.xml.dist  --no-coverage --colors tests/UserAgentsTest/V4/FullTest.php`

File: tests/issues/issue-2567.php [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    'issue-2567-E' => [
+        'ua' => 'Mozilla/5.0 (iPad; CPU OS 15_3 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.3 Mobile/15E148 Safari/604.1',
+        'properties' => [
+            "Platform"=> "ipadOS",
+            'Platform_Version' => '15.3',
+            'Platform_Maker' => 'Apple Inc',
+            "Browser_Version" => "Safari",
+            "Browser_Version" => "15.3",
+            "Comment" => "Mobile Safari 15.3",
+            "Device_Name" => "iPhone",
```
Comment (mimmi20) [inline tests/issues/issue-2567.php]: This must be the `iPad`. This test is failing

**Review by mimmi20 — CHANGES_REQUESTED** [review]

**Review by mimmi20 — APPROVED** [review]

**Comment by willyaranda** [comment]
Feel free to merge @mimmi20

**Comment by asgrim** [comment]
Thanks @willyaranda

**Pull request merged**